lib/camp.js

```diff
@@ -172,6 +173,22 @@ function getCompressedStream(ask) {
   return res;
 }
 
+// Concatenate an array of streams into a single stream.
+function concatStreams(array) {
+  var concat = new stream.PassThrough();
+
+  function pipe(i) {
+    var end = i === array.length - 1;
+    array[i].pipe(concat, {end: end});
+    if (!end) {
+      array[i].on('end', pipe.bind({}, i + 1));
```

Comment:
I don't think so, can you show me a benchmark that shows that using bind slows routed requests down significantly?

Also, how would you write this without bind?